app/cdap/components/WrangleHome/Components/OngoingDataExploration/types.ts

```diff
@@ -78,3 +78,12 @@ export interface IMassagedObject {
   subText: string;
   workspaceId: string;
 }
+
+export interface IEachData {
```

Comment:
Can we get a more descriptive type name for this

Reply:
Done

Reply:
updated to IOngoingExplorationCard